certificate-policy.md

```diff
@@ -1,46 +1,43 @@
-**U.S. PKI**
-
+** This is a DRAFT work in progress.  Please see the source repository [here](https://github.com/uspki/policies) This is being developed in the open and contributions are welcome.**
 
+**Federal PKI**
 
 **Certificate Policy**
 **for the**
 **Issuance and Management of**
-**Publicly-Trusted Certificates**
+**Publicly-Trusted Non-Person Entity (NPE) Certificates**
 
 
 
 
 **Version 0.0.1**
 
-**October 28, 2016**
+**TODO: DATE**
 
-For details on copyright and license information for this doucment, see section 1.1.
+For details on copyright and license information for this document, see section 1.1.
 
 
 # 1. INTRODUCTION
 
 ## 1.1 Overview
-This CP describes an integrated set of technologies, protocols, identity-proofing, lifecycle management, and auditing requirements that are necessary (but not sufficient) for the issuance and management of Publicly-Trusted Certificates; Certificates that are trusted by virtue of the fact that their corresponding Root Certificate is distributed in widely-available application software. The requirements are not mandatory for Certification Authorities unless and until they become adopted and enforced by relying-party Application Software Suppliers.
+This CP describes an integrated set of technologies, protocols, identity-proofing, lifecycle management, and auditing requirements that are necessary (but not sufficient) for the issuance and management of US Federal issued Publicly-Trusted Certificates; Certificates that are trusted by virtue of the fact that their corresponding Root Certificate is distributed in widely-available application software. The requirements are not mandatory for Certification Authorities unless and until they become adopted and enforced by relying-party Application Software Suppliers.
```

Comment:
What is the intent of "(but not sufficient)" and the "subset of requirements" in line 27 below.  Will there be some additional document that fills in the rest?  My preference - the CP should be as complete as possible in order to ensure consistent implementation.